### ISSUE TITLE
Calendar "onMonthLoaded" not being triggered after clicking on month/year list

### ISSUE THREAD

**Issue by shamank** [acidb/mobiscroll#336]
It works as expected if tapping or touching month/year handler buttons, but not if you select from the list that appears if you touch the month/year label. This is the only method I found for doing what I need to do (attach an event every time the month/year change) :( is there any workaround for this? Thanks!

Example:
```
$(el).mobiscroll().calendar({
    onMonthLoaded: function () { console.log("ADASD"); }
});
```
![yes](https://cloud.githubusercontent.com/assets/819192/8860768/5412721c-315c-11e5-9dc5-7abe44dadab6.png)

![no](https://cloud.githubusercontent.com/assets/819192/8860782/67e0427e-315c-11e5-8a60-57313be451d7.png)

**Comment by dioslaska** [comment]
Thanks for the report, I was able to reproduce it.
The onMonthChange event however fires correctly. Maybe you can use that for what you are doing.

**Comment by shamank** [comment]
Yes, but the onMonthChange event doesn't fire if you change the year (current month is refreshed according to the new year selected but the onMonthChange event is not fired). Maybe because the triggering condition is waiting for a change of name. Thanks.

**Comment by dioslaska** [comment]
What version are you using? I'm testing with 2.16.1 and the onMonthChange event fires when changing the year as well.

**Comment by shamank** [comment]
Yes, sorry, you are right (2.16.1). The problem is that when changing the year and firing the event, in that moment the datagrid is not yet loaded (as opposite, it is when changing the month). I'm attaching material design events to buttons (via js) so I need to attach them after the buttons were already parsed by JQM.

Debugger paused in console.log():
![1](https://cloud.githubusercontent.com/assets/819192/8869747/f6b3b3bc-31bd-11e5-89bf-e387f7dc69d1.png)

Resuming:
![2](https://cloud.githubusercontent.com/assets/819192/8869754/0dbf82a2-31be-11e5-94aa-37e6398955b0.png)

**Comment by dioslaska** [comment]
Here's a workaround until it gets fixed. The navigate method needs to be overridden / extended to trigger the onMonthLoaded method:

        $('#calendar').mobiscroll().calendar({
            onMonthLoaded: function () {
                console.log('load');
            },
            onInit: function (inst) {
                if (!inst._navigate) {
                    inst._navigate = inst.navigate;

                    inst.navigate = function (d, anim) {
                        var s = inst.settings;
                            y = s.getYear(d),
                            m = s.getMonth(d),
                            visible = inst.isVisible();

                        // Call original function
                        inst._navigate(d, anim);

                        if (!anim && visible) {
                            inst.trigger('onMonthLoaded', [y, m]);
                        }
                    };
                }
            }
        });

**Comment by shamank** [comment]
Thank you!

**Comment by dioslaska** [comment]
Fixed, will be released in 2.17.0
